docs/products/kafka/howto/manage-acls.md

```diff
@@ -1,77 +1,452 @@
 ---
-title: Manage users and access control lists
+title: Manage access control lists in Aiven for Apache Kafka®
 ---
 
-Aiven for Apache Kafka® uses access control lists (ACL) and user definitions to grant specific rights for producing or consuming topics.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-For detailed information on permission mapping,
-see [Access control lists and permission mapping](/docs/products/kafka/concepts/acl).
+Access control lists (ACLs) in Aiven for Apache Kafka® define permissions for topics, schemas, consumer groups, and transactional IDs.
+ACLs control which authenticated users or applications (principals) can perform specific
+operations on these resources.
 
-To manage users and ACL entries, access the corresponding sections in
-the left sidebar of the service page within the [Aiven
-Console](https://console.aiven.io/).
+## Types of ACLs
 
-## Add a user
+Aiven for Apache Kafka® supports two types of ACLs:
 
-:::warning
-A user with the `Admin` permission can create topics with any name, as
-the `CreateTopics` permission is applied at the cluster level.
+- **Aiven ACLs**: These provide topic-level permissions and support wildcard patterns.
+- **Kafka-native ACLs**: These offer advanced, resource-level permissions with `ALLOW`
+  and `DENY` rules for operations on multiple resource types, including topics, groups,
+  and clusters.
 
-All other permissions related to a topic (`Alter`, `Delete`) **only**
-apply to the topics matching the pattern that you specify.
+:::note
+ACL restrictions currently do not apply to Kafka REST. Rules are applied based on the
+username and topic names, but there are no restrictions on consumer group names.
+
+Support for extending these restrictions to Kafka REST is under development.
```

Comment:
There is a flag on service's user config if ACLs apply to Kafka REST or not, but by default it has been false. However, if enabled, then all ACLs apply same way on KAfka REST. I don't think there is any under development features here.

Reply:
Can you confirm the exact name of the flag in the service's user configuration that determines if ACLs apply to Kafka REST?
The "under development features" were mentioned in the previous content.

Reply:
It is `kafka_rest_authorization` https://aiven.io/docs/products/kafka/karapace/howto/enable-kafka-rest-proxy-authorization

Reply:
Ahh, I see, thanks.

Reply:
Updated the note and cross-linked content.